documentation/example.yml

```diff
@@ -19,6 +19,8 @@ Network:
 OS:
   os: "debian"
   version: 12
+  version-name: bookworm
```

Comment:
I like target architecture, but we shouldn't make the user specify the version and the version-name. Since nothing else uses version right now to my knowledge, I think its safe to just replace the number with the name

Reply:
Addressed in commit [8492ef9](https://github.com/gingrm02/genisys/pull/16/commits/8492ef912e5e743fe700ee97c91a2be601d35cdc)